src/tictactoe_clojure/board.clj

```diff
@@ -1,28 +1,82 @@
-(ns tictactoe-clojure.board)
+(ns tictactoe-clojure.board
+  (:require [tictactoe-clojure.console :as console])
+  (:require [clojure.math.numeric-tower :as math]))
 
-(def new-board [0 1 2 3 4 5 6 7 8])
+(defn get-size
```

Comment:
I think that's better. It's good practice to ask yourself to list out the responsibilities. What do you think the responsibilities were?